editor/shader/shader_editor_plugin.cpp

```diff
@@ -859,6 +830,16 @@ void ShaderEditorPlugin::_notification(int p_what) {
 	}
 }
 
+void ShaderEditorPlugin::shortcut_input(const Ref<InputEvent> &p_event) {
```

Comment:
This doesn't seem to have any effect. Not sure what has to be focused to catch the shortcut.

EDIT:
Or maybe it's because you didn't enable shortcut input 🙃